docs/contribute/where-to-work.md

```diff
@@ -0,0 +1,99 @@
+
+# **How do I know where to contribute?**
```

Comment:
> # **How do I know where to contribute?**

A very brief drive-by comment that this content seems like it should be integrated with or replace what's in https://docs-v3-preview.elastic.dev/elastic/docs-builder/tree/main/contribute/#contribute